cfme/cloud/provider/__init__.py

```diff
@@ -132,6 +132,7 @@ def step(self):
         self.prerequisite_view.navigation.select('Compute', 'Clouds', 'Providers')
 
     def resetter(self):
+        self.appliance.browser.widgetastic.browser.refresh()
```

Comment:
This has been moved to #6836 for both cloud and infra, leaving in here for now since it cleans up PRT considerably.